.github/CODEOWNERS

```diff
@@ -2,8 +2,8 @@
 * @keitwb @jcheng-splunk @mstumpfx @asuresh4 @rmfitzpatrick @signalfx/gdi-data-collection-maintainers
 
 # Docs reviewers
-*.md @signalfx/docs @signalfx/gdi-data-collection-approvers
-*.rst @signalfx/docs @signalfx/gdi-data-collection-approvers
-docs/ @signalfx/docs @signalfx/gdi-data-collection-approvers
-README* @signalfx/docs @signalfx/gdi-data-collection-approvers
+*.md @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/docs
+*.rst @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/docs
+docs/ @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/docs
+README* @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/docs
```

Comment:
signalfx/docs is already the the first reviewer specified

Reply:
Ah, sloppy dupe here. Hang on a sec.

Reply:
It's OK anyway, no? Precedence is not required here; we only aim at avoiding that tech writers have exclusive approval powers over those files.

Reply:
Yeah the CODEOWNERS file is already set up this way - https://github.com/signalfx/signalfx-agent/blob/main/.github/CODEOWNERS

Your PR is just moving the @signalfx/docs team behind the @signalfx/gdi-data-collection-approvers which is functionally the samee

Reply:
Oh, OK. I get it now. This is not the repo where you rolled back the changes the other day. My fault. Closing this one now. :)

Reply:
No problem - If you are referencing this: https://github.com/signalfx/signalfx-agent/pull/1883#issuecomment-908470388 
That's actually the PR where we made the change so that both teams could approve so we wouldn't block each other.